src/components/Topbar/StudioTopbar/StudioTopbar.tsx

```diff
@@ -101,11 +103,30 @@ const StudioTopbar: React.FC = () => {
   return (
     <>
       <StyledTopbarBase variant="studio">
-        <StudioContainer>
+        <StudioTopbarContainer>
           <Button icon="add-video" />
-          <ChannelInfo channel={currentChannel} member={member.name} onClick={handleDrawerToggle} />
+          {member.channels.length ? (
+            <ChannelInfo channel={currentChannel} member={member.name} onClick={handleDrawerToggle} />
+          ) : (
+            <>
+              <NewChannelButtonLink
+                icon="new-channel"
+                variant="tertiary"
+                size="large"
+                to={routes.studio.newChannel()}
+              />
+              <StyledLink to={routes.studio.newChannel()}>
```

Comment:
We have both the `ButtonLink` and `StyledLink` here, do we need both?

Reply:
I also don't think this needs to be a link. If there are no channels, we will force the user go through the create channel flow